turbo.json

```diff
@@ -33,8 +33,7 @@
         "PIE_URL_PREFIX"
       ],
       "dependsOn": [
-        "^build",
-        "@justeattakeaway/pie-icons#build"
```

Comment:
Now that pie-icons is defined in the pie-docs/package.json, we don't need to define this dependency anymore.

This was needed because previously, Turborepo didn't know pie-icons was part of the build chain for pie-docs, as it was a pie-monorespo dependency